lib/_tls_common.js

```diff
@@ -240,7 +240,7 @@ exports.translatePeerCertificate = function translatePeerCertificate(c) {
     c.infoAccess = Object.create(null);
 
     // XXX: More key validation?
-    info.replace(/([^\n:]*):([^\n]*)(?:\n|$)/g, function(all, key, val) {
+    info.replace(/([^\n:]*):([^\n]*)(?:\n|$)/g, function replacerCallback(all, key, val) {
```

Comment:
Is that supposed to be `replaceCallback` instead of `replacerCallback`?

Reply:
It wasn't a typo in my head. I suppose that I makes sense in both versions:

- `replacerCallback` -> the replacer function (inside `replace`)
- `replaceCallback` -> the callback of the `replace` function

Happy to change it of course

Reply:
Nah, if it wasn't a typo, it's fine with me. Just wanted to give you the opportunity to fix it up if it was a mistake.